### ISSUE TITLE
Crash on iOS "transformedRect...invalid transform"

### ISSUE THREAD

**Issue by ou2s** [archriss/react-native-image-gallery#53]
Hi, we released an app with this library on production. Our crash reporting tool is warning us about some crashes on iOS related to this library.

Here are the logs:
`Error: transformedRect...invalid transform`

<img width="943" alt="screen shot 2018-01-08 at 20 48 30" src="https://user-images.githubusercontent.com/688029/34689372-9327b268-f4b5-11e7-858f-2d16abeec512.png">

Do you guys have an idea? I have trouble reproducing the crash.

The crashes occurred on two devices:
- iPad Mini  iOS 9.3.5
- iPad 4th Gen iOS 8.0.2

**Comment by msageryd** [comment]
Can you reproduce the problem and get hold of some more information? The following would be helpful:
- What did the user try to do before the crash?
- What is the value of transform? It should be three numbers, {scale, translateX, translateY}

**Comment by ou2s** [comment]
@msageryd not able to reproduce the crash, but I will add some reporting events for the values you mentioned. Do you think this may be related to orientation? Thanks for the help.

**Comment by msageryd** [comment]
What version of RN are you running? I read in the changelog for 0.52 RC1 that something changed with `measure()`. Measure() is used by **ViewTransformer** when the device orientation changes. It's a long shot and I actually don't think this is the problem.

Get back here as soon as you have some more data.

**Comment by ou2s** [comment]
Unfortunately, we were not able to collect data and that crash is impacting us too much. Tried many things to reproduce it and also to fix the JS code but with no luck. 

Because it seems like we are the only one having this crash, it might be related to the fact that we don't display the Gallery until react-navigation has finished its transition. Sadly, we might have to drop this nice library in our app because of this bug until we changed react-navigation for wix-navigation.
